samples/SkiaAndroidSample/src/main/java/org/jetbrains/skiko/sample/Clocks.kt

```diff
@@ -152,20 +159,23 @@ class Clocks(private val layer: SkiaLayer): SkikoView {
             when (event.key) {
                 SkikoKey.KEY_NUMPAD_ADD -> scale *= 1.1
                 SkikoKey.KEY_I -> {
-                    if (event.modifiers == SkikoInputModifiers.CONTROL) {
+                    if (event.modifiers == SkikoInputModifiers.SHIFT) {
```

Comment:
Why this change?

Reply:
> refactored [Clocks] example due to emulator shortcuts
Android emulator uses CONTROL key to emulate two-finger-touch